gnosis/safe/safe.py

```diff
@@ -1185,3 +813,453 @@ def send_multisig_tx(
         )
 
         return EthereumTxSent(tx_hash, tx, None)
+
+
+class SafeV130(SafeCommon):
+    @cached_property
+    def contract(self) -> Contract:
+        return get_safe_V1_3_0_contract(self.w3, address=self.address)
```

Comment:
If we add `contract_fn = get_safe_V0_0_1_contract` for class SafeV001 or the same for other classes is going to fail when we call `self.contract_fn` because `get_safe_V0_0_1_contract` is not part of safe class and we are passing the instance.